medusa/init/__init__.py

```diff
@@ -38,12 +42,21 @@ def _check_python_version():
         sys.exit(1)
 
 
-def _lib_location():
-    return os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', 'lib'))
+def _remove_dead_python_bytecode():
```

Comment:
Please remove this method.
1. It's unused.
2. It removes files when the `medusa` package is first imported (per python session), that is not a good idea IMO.
	(you already have an item in your to-do list to update the `UpdateManager` code as I suggested)

Reply:
done